### ISSUE TITLE
Evaluate the current POC pipeline for operators

### ISSUE THREAD

**Issue by JaimeMagiera** [okd-project/operators#3]
We have created a POC Tekton pipeline https://github.com/okd-project/okd-operator-pipeline

It contains the relevant artifacts to build a generic container image with all the support tools to compile, build operators , update/check bundles , create bundles, indexes etc and also add the artifacts to  a new/existing catalog

We have also created an example operator https://github.com/okd-project/okd-sample-operator to show case the operator pipeline

Basically the thinking here is to use/not use all or sections of the pipeline (project)
